src/main/kotlin/uk/gov/dluhc/printapi/database/repository/TemporaryCertificateRepository.kt

```diff
@@ -10,6 +10,8 @@ import java.util.UUID
 @Repository
 interface TemporaryCertificateRepository : JpaRepository<TemporaryCertificate, UUID> {
 
+    fun findByGssCodeAndSourceTypeAndSourceReference(gssCode: String, sourceType: SourceType, sourceReference: String): TemporaryCertificate?
+
```

Comment:
Looks like there isn't a TemporaryCertificateRepositoryIntegrationTest class, was that a conscious decision because it's covered in RestIntTest?

Reply:
That's right (except I'm relying on `FinalRetentionPeriodDataRemovalJobIntegrationTest`). I'm not keen on having that many ITs in general, but also this method is a copy & paste from the CertificateRepository and it seemed a bit unnecessary. I'm relying on `FinalRetentionPeriodDataRemovalJobIntegrationTest` to expose any regression issues.